pyannote/audio/pipelines/clustering.py

```diff
@@ -200,6 +232,19 @@ def __init__(self, metric: str = "cosine", expects_num_clusters: bool = False):
             ["Affinity", "Unnormalized", "RandomWalk", "GraphCut"]
         )
         self.eigengap = Categorical(["Ratio", "NormalizedDiff"])
+        self.spectral_min_embeddings = LogUniform(1, 100)
```

Comment:
My understanding is that `spectral_min_embeddings` is supposed to be an integer, right? 
`LogUniform` makes it a `float` and artificially increases the dimension of the search space. How critical is this hyperparameter? Can we not make it constant to, say, 5?
I'd rather avoid adding almost useless hyperparameters to the search space.

Reply:
Makes sense. Changed to `Categorical([5, 10])`